lib/puppet/type/f5_passwordpolicy.rb

```diff
@@ -0,0 +1,73 @@
+Puppet::Type.newtype(:f5_passwordpolicy) do
+  @doc = 'Sets the password policy on the BIG-IP system.'
+
+  apply_to_device if Facter.value(:url).nil?
```

Comment:
The `if Facter.value(:url).nil?` makes it possible to use `puppet resource`  or `puppet apply` to set properties.  This is really helpful during testing! :)

Reply:
+1 @alexjfisher